SYCL/Sampler/unnormalized-clamp-linear.cpp

```diff
@@ -0,0 +1,173 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
+// RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+// XFAIL: gpu && (level_zero || opencl || cuda)
```

Comment:
```suggestion
// XFAIL: gpu
```
The test failing on GPU for all supported BEs. There is no reason to list all of them

Reply:
done